### DISCUSSION TITLE
Return to initial group

### DISCUSSION THREAD

**Discussion by aloispichler** [qtile/qtile#3926] [Q&A] [answered]
Assume the current focus is on group 3.
Hitting $mod+8 moves to group 8.
Hitting $mod+8 _again_ should return to group 3.
Can this be accomplished in qtile? I was using this feature a lot in i3.

**Comment by elParaguayo** [comment, marked as answer]
Yes. This is possible in qtile.

You've probably got a line something like
```python
lazy.group[i.name].toscreen()
```
Change this to:
```python
lazy.group[i.name].toscreen(toggle=True)
```

**Reply by aloispichler** [reply]
I missed that. Wonderful. And thank you for the prompt response!
